README.md

```diff
@@ -4,11 +4,17 @@ Repository with useful scripts for creating PKI, building and starting Verify ap
 
 ## Before you start
 
-You'll need to install [cfssl](https://github.com/cloudflare/cfssl) which is used to generate the PKI.
+If you don't already have Go, you'll need to install it.
```

Comment:
I'm not sure this is necessary if you're installing via Homebrew. The scripts run `brew install cfssl` if the user doesn't have it installed.